meeting_notes/20171125_planning-meeting-notes.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Meeting Notes - November 25, 2017
+location: Semaphore Studio 307, Claude T. Bissell
+attendees: 9
+date: 2017-11-25
```

Comment:
Should this be the date we held the meeting, or publish date of this file? Same thought about the file name

Reply:
I think date of meeting

Reply:
changed.